### PR TITLE
[Storybook] Fixes related to grey 25 as default background color

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

Fixes related to grey 25 as default background color

-----

image

-----

### CODE REVIEW

File: stories/documentation/integration/utilities/shadows.stories.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ const Template: StoryFn<ShadowsStory> = (args: ShadowsStory) => ({
 	styles: [
 		`
 		div {
+			background-color: #FFFFFF;
```
Comment (vvalentin-lucca) [inline stories/documentation/integration/utilities/shadows.stories.ts:26]: `--colors-white-color` ?

File: stories/documentation/structure/box/box-arrow.stories.ts [hunk 2]
```diff
@@ -104,6 +104,13 @@ function getTemplate(args: ArrowBasicStory): string {
 const Template: Story<ArrowBasicStory> = (args: ArrowBasicStory) => ({
 	props: args,
 	template: getTemplate(args),
+	styles: [
+		`
+		:host {
+			display: block;
+		}`,
+		args.grey === true ? ':host { background-color: white; margin: -15px -15px; padding: 15px 15px; }' : '',
```
Comment (vvalentin-lucca) [inline stories/documentation/structure/box/box-arrow.stories.ts:112]: Why not just `1rem`?

Comment (jeremie-lucca) [reply stories/documentation/structure/box/box-arrow.stories.ts:112]: These values are related to SB spacings.

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2523/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/12094/ui-diff)

**Pull request merged**